src/main/java/heat/config/DatabaseConfig.java

```diff
@@ -5,21 +5,32 @@
 import org.springframework.context.annotation.Configuration;
 
 import javax.sql.DataSource;
+import java.net.URI;
 import java.net.URISyntaxException;
 
 @Configuration
 public class DatabaseConfig {
+
+    private static DataSource dataSource;
+
     @Bean
     public DataSource dataSource() throws URISyntaxException {
-        String dbUrl = System.getenv("JDBC_DATABASE_URL");
-        String username = System.getenv("JDBC_DATABASE_USERNAME");
-        String password = System.getenv("JDBC_DATABASE_PASSWORD");
+        URI dbUri = new URI(System.getenv("DATABASE_URL"));
+
+        String username = dbUri.getUserInfo().split(":")[0];
+        String password = dbUri.getUserInfo().split(":")[1];
+        String dbUrl = "jdbc:postgresql://" + dbUri.getHost() + ':' + dbUri.getPort() + dbUri.getPath();
 
         HikariDataSource basicDataSource = new HikariDataSource();
         basicDataSource.setJdbcUrl(dbUrl);
         basicDataSource.setUsername(username);
         basicDataSource.setPassword(password);
+        dataSource = basicDataSource;
```

Comment:
Why do we need this? Can we use (mb just inject?) the bean later to get the data source? Please, leave a comment with explanation in case you find this solution reasonable.

Reply:
This line is here, because I don't know, how to use DI in this case. I'll be glad to hear it from you.

Reply:
No idea. Leave it as is.

Reply:
Done.